### PR TITLE
Upgrade n-makefile, pass responsiblity for building (but not minifying) JS+CSS to n-makefile AND eliminate the need for NPM_BIN_ENV everywhere

### PR DESCRIPTION
**Pull request by matthew-andrews**

### CODE REVIEW

File: n.Makefile [hunk 1]
```diff
@@ -30,6 +30,17 @@ deplo%: _deploy_apex
 verif%: _verify_lintspaces _verify_eslint _verify_scss_lint
 	@$(DONE)
 
+# build (includes build-production)
+buil%:
+	@$(warning WARNING: Work in progress, build-production does not yet minify.  Use with caution.)
+	@if [ -e webpack.config.js ]; then $(NPM_BIN_ENV) && webpack $(if $(findstring build-production,$@),--bail,--dev); fi
```
Comment (matthew-andrews) [inline n.Makefile]: :see_no_evil:

**Comment by matthew-andrews** [comment]
0.0.63 covers the polyfills work^^

**Comment by matthew-andrews** [comment]
0.0.64 to undo the polyfills work^^

**Comment by ironsidevsquincy** [comment]
:+1:

**Pull request merged**